packages/thirdweb/src/wallets/connection/autoConnectCore.ts

```diff
@@ -183,13 +183,18 @@ const _autoConnectCore = async ({
         wallet,
         client: props.client,
         lastConnectedChain,
-        authResult: result?.authResult,
+        authResult: urlToken?.authResult,
       });
       manager.addConnectedWallet(wallet);
     } catch {
       // no-op
     }
   }
+
+  // Auto-login with SIWE
+  if (urlToken && activeWallet && props.siweLogin) {
```

Comment:
I think we need to check if it actually requires auth here too. I think you can pass the require auth function down as well maybe?